admin-app/src/components/GameList.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { List, Datagrid, TextField, DateField, NumberField } from 'react-admin';
+
+interface GameListProps {}
+
+export const GameList = (props: GameListProps) => (
+  <List {...props}>
+    <Datagrid rowClick="edit">
+      <TextField source="id" />
+      <DateField source="createdAt" />
+      <DateField source="updatedAt" />
+      <NumberField source="player1.id" />
+      <NumberField source="player2.id" />
```

Comment:
Can you confirm you did plan to add the 'status' column later?

Reply:
Yep, but i can add it now too !